core/src/test/scala/org/broadinstitute/dsde/rawls/mock/MockWorkspaceManagerDAO.scala

```diff
@@ -13,6 +13,9 @@ import java.util.UUID
 import scala.collection.concurrent.TrieMap
 import scala.jdk.CollectionConverters._
 
+object MockConstants {
+  final def storageAccountId: UUID = UUID.fromString("11111111-2222-3333-5555-666666666666")
```

Comment:
You can make this a val and forego the final qualifier:

```suggestion
  val storageAccountId: UUID = UUID.fromString("11111111-2222-3333-5555-666666666666")
```

This raises the question though, why does this need to be a constant?

Reply:
The issue is that the account ID passed back from creating the storage account is what will be passed to the creation request for the container. If I don't make it a constant, I don't have a way to verify that the right thing was passed into the container creation.

See https://github.com/broadinstitute/rawls/pull/1857/files#diff-b8196c4af84188faa6eb1f65595077eb0b25f6c06f5a8e64b5c60e4832b93118R198

Reply:
You could override the `MockWorkspaceManagerDAO` to return a specified storage account id. Something like this should work:

```
val storageAccountId = UUID.randomUUID()
val customWsmDao = new MockWorkspaceManagerDAO() {
  override def mockCreateAzureStorageAccountResult() = new CreatedControlledAzureStorage().
    resourceId(storageAccountId).azureStorage(new AzureStorageResource()) 
}
val workspaceManagerDAO = Mockito.spy(customWsmDao)
```

Reply:
Another alternative would be to pass it as a parameter to `MockWorkspaceManagerDAO` and have it default to a random UUID

Reply:
Thanks @marctalbott, this is a much nicer approach.